### PR TITLE
Research documentation

### PR DESCRIPTION
**Pull request by sarahmhgov**
## What does this PR do? 🛠️
Adds a page of documentation on research to date

## What does the reviewer need to know? 🤔
Shouldn't touch anything but the one doc!

### CODE REVIEW

File: docs/research.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<h1> How has research been involved in informing the weather.gov 2.0 design? </h1>
```
Comment (colinmurphy01) [inline docs/research.md:1]: For future reference. You don't need to use html markup for headings.

[hash symbols](https://www.markdownguide.org/basic-syntax/#headings) can be used instead

File: docs/research.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| WFO onsite - Nashville | Nov 2023 | Better understand partner experiences before, during and after a weather hazard | Identified partner painpoints during a hazard - post-event support, mapping and radar, etc |
+| R6 - Content Strategy testing | June 2024 | Understand what supplementary information users need beyond forecasts, and how they want to find and use them | Hypothesis of how to organize content, by locations that are relevant to decision maker areas.  |
+
+
+<h3> General public usability research </h3>
+
+| Name  | When  | Interest  | Usability| Relevance| Comprehension|
+| -------- | ------- | -------- | ------- |------- | -------- |
+| R1 - Usability (public) | Dec 2024 | ------- |------- | -------- |
+| R2 - Usability (public) | Jan 2024 |  Most participants said they would try out weather.gov alongside other apps and website | Some participants had some issues finding what they were looking for| Additional details required | Issues interpreting metrics |
```
Comment (colinmurphy01) [inline docs/research.md:27]:
```suggestion
| R2 - Usability (public) | Jan 2024 |  🟢 Most participants said they would try out weather.gov alongside other apps and website | 🟡 Some participants had some issues finding what they were looking for | 🟡 Additional details required | 🟡 Issues interpreting metrics |
```

Comment (colinmurphy01) [reply docs/research.md:27]: I would love maintaining visual indicators in this table if possible.

- 🟢 
- 🟡 
- 🔴 
- ⚪  (for open question/uncertain use cases?)

File: docs/research.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| WFO onsite - Sterling (DC) | July 2023 | Onsite tour + observations at Sterling WFO office (DC area). Includes in-depth notes on internal tools and systems (AWIPs, etc) | Understanding of current forecaster workflow and tools in creating forecasts and various types of content  |
+| Forecaster research | Sept 2023 | Validate our understanding of the forecaster experience before, during and after a weather hazard | Began to identify some common needs for a CMS experience |
+| WFO onsite - Nashville | Nov 2023 | Better understand partner experiences before, during and after a weather hazard | Identified partner painpoints during a hazard - post-event support, mapping and radar, etc |
+| R6 - Content Strategy testing | June 2024 | Understand what supplementary information users need beyond forecasts, and how they want to find and use them | Hypothesis of how to organize content, by locations that are relevant to decision maker areas.  |
+
+
+<h3> General public usability research </h3>
+
+| Name  | When  | Interest  | Usability| Relevance| Comprehension|
```
Comment (colinmurphy01) [inline docs/research.md:24]: Can we add one more column with a link to the summary presentation hosted in the NOAA drive for easy lookup?

**Review by colinmurphy01 — APPROVED** [review]
I have some additional changes that could make this a little more helpful.

But all in all, this is really great and I'm excited for this to be included in our github docs.

Once published we can then link to it in other docs.

File: docs/research.md (hunk 3 above)
Comment (sarahmhgov) [reply docs/research.md:24]: yep! I didn't because I thought we discussed not doing that, but happy to

Comment (sarahmhgov) [reply docs/research.md]: added in

**Comment by colinmurphy01** [comment]
Chatted with eng about the test failure. We should be good to merge.

**Pull request merged**